ui/package.json

```diff
@@ -3,21 +3,22 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.9.0",
     "@elastic/datemath": "5.0.3",
-    "@elastic/eui": "32.3.0",
-    "@gojek/mlp-ui": "1.5.5",
+    "@elastic/eui": "63.0.0",
+    "@gojek/mlp-ui": "1.6.0",
     "@overgear/yup-ast": "^1.0.3",
     "@reach/router": "1.3.4",
     "@sentry/browser": "5.15.5",
     "js-yaml": "^4.0.0",
     "mitt": "^2.1.0",
-    "moment": "^2.29.0",
+    "moment": "^2.29.4",
     "object-assign-deep": "^0.4.0",
     "query-string": "^7.0.1",
-    "react": "16.14.0",
+    "react": "17.0.2",
     "react-collapsed": "^3.0.1",
     "react-diff-viewer": "^3.1.1",
-    "react-dom": "16.14.0",
+    "react-dom": "17.0.2",
     "react-lazylog": "git+https://github.com/gojekfarm/react-lazylog#master",
     "react-scroll": "^1.7.16",
     "react-scroll-to-bottom": "^4.0.0",
```

Comment:
@romanwozniak,  please also update `react-scripts` to the stable version: ref https://github.com/gojek/mlp/pull/55

Reply:
I meant you, @deadlycoconuts 😅